docs/source/tutorials/odsc_tutorial/getting_started.rst

```diff
@@ -170,3 +170,9 @@ The most important fields are:
 Also, note the role of ``**kwargs`` in the function signature, which allows
 to overwrite any of the default values (e.g., for ``max_wallclock_time``,
 ``n_workers``, or ``instance_type``) with command line arguments.
+
+.. note::
+   In the Syne Tune experimentation framework, a tuning problem (i.e., training and
+   evaluation script together with defaults) is called a *benchmark*.
+   This terminology is used even if the goal of experimentation is not benchmarking
+   (i.e., comparing different HPO methods), as is the case in this tutorial here.
```

Comment:
After this refactor is completed, what will remain in the top-level `benchmarking/` folder, and what is the purpose of it? (aside from `benchmarking/nursery`) 

And a question arises: what is the difference between a benchmark and a study?

My understanding of existing terminology: 
- `experiment`: a single run of HPO
- `study`: a set of experiments (e.g., different HPO methods, different config spaces, different seeds)

By the existing naming scheme, is a benchmark an implementation of a study?

Reply:
There are two purposes of `benchmarking` folder:

* Collect training codes for benchmarks (in `benchmarking/training_scripts`)
* Collect examples for benchmarking studies (in `benchmarking/examples`, `benchmarking/nursery`)

Reply:
Naming:

* A "benchmark" is a tuning problem, consisting of a training code or surrogate blackbox, plus a default config space
* An experiment is a single run of HPO, configured by a benchmark (tuning problem), a method, a seed, etc
* A study is a set of experiments, designed to answer some statistical question or gather insights
* Some studies are about "benchmarking",  meaning comparing how different HPO methods compare on one or more benchmarks
* Other studies are about something else, so for example use a single HPO method only